src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -46,9 +51,15 @@ class Point(int x, int y);
                 // (2,12): error CS8652: The feature 'records' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
                 // class Point(int x, int y);
                 Diagnostic(ErrorCode.ERR_FeatureInPreview, "(int x, int y)").WithArguments("records").WithLocation(2, 12),
-                // (2,12): error CS8761: Records must have both a 'data' modifier and parameter list
+                // (2,12): error CS8800: Records must have both a 'data' modifier and non-empty parameter list
                 // class Point(int x, int y);
                 Diagnostic(ErrorCode.ERR_BadRecordDeclaration, "(int x, int y)").WithLocation(2, 12),
+                // (2,17): error CS0518: Predefined type 'System.Runtime.CompilerServices.IsExternalInit' is not defined or imported
```

Comment:
Is there an expectation that eventually the compiler will synthesize an internal version of this attribute if needed, similar to IsReadOnlyAttribute?

Reply:
I asked this question as well: no, because there are various ordering problems with synthesizing types used in custom modifiers.